### PR TITLE
added some tests

### PR DESCRIPTION
**Pull request by dariusdev**
test, community holders from 4 to 7, naming alignment

### CODE REVIEW

File: contracts/DevTokensHolder.sol [hunk 1]
```diff
@@ -7,9 +7,9 @@ import "./ERC20.sol";
 contract DevTokensHolder is Controlled {
   using SafeMath for uint256;
 
-  uint256 collectedTokens;
-  Contribution contribution;
-  ERC20 aix;
+  uint256 public collectedTokens;
+  Contribution private contribution;
+  ERC20 private aix;
```
Comment (rstormsf) [inline contracts/DevTokensHolder.sol]: why this should be `private`

Comment (dariusdev) [reply contracts/DevTokensHolder.sol]: just added. i will revert

**Review by rstormsf — APPROVED** [review]

File: contracts/CommunityTokenHolder.sol [hunk 2]
```diff
@@ -22,7 +22,7 @@ contract DevTokensHolder is Controlled {
     uint256 balance = aix.balanceOf(address(this));
     uint256 total = collectedTokens.add(balance);
     // This wallet will get a 29% of the total tokens.
-    // since scaling 4 of 29 to a percentage looks horrible (13.7931034483),
+    // since scaling 7 of 29 to a percentage looks horrible (24.1379310344828),
```
Comment (Janther) [inline contracts/CommunityTokenHolder.sol]: 👍 great commitment on updating also the comments

**Review by Janther — APPROVED** [review]

File: contracts/CommunityTokenHolder.sol (hunk 2 above)
Comment (dariusdev) [reply contracts/CommunityTokenHolder.sol]: 👍 :)

**Pull request closed without merging**